core/include/detray/masks/annulus2.hpp

```diff
@@ -65,7 +65,15 @@ class annulus2 final
         std::numeric_limits<scalar>::epsilon()};
 
     /* Default constructor */
-    annulus2() = default;
+    annulus2() {
```

Comment:
Is it possible to do this via a constructor initializer list?

Reply:
Can you elaborate?

Reply:
I mean something like this:
```
    annulus2() : base_type({0.,
                         std::numeric_limits<scalar>::infinity(),
                         -std::numeric_limits<scalar>::infinity(),
                         std::numeric_limits<scalar>::infinity(),
                         0.,
                         0.,
                         0.}, {}) {}
```